workers/cloudflare-workers/index.test.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="@types/jest" />
+
+import fetch from 'node-fetch'
+
+describe('use data proxy', () => {
+  test('fetch response', async () => {
+    jest.setTimeout(30000)
+
+    const response = await fetch(process.env.DEPLOYMENT_URL!)
```

Comment:
to make it more explicit what this is:
```suggestion
    const cloudflareWorkersDeploymentUrl = process.env.DEPLOYMENT_URL!
    const response = await fetch(cloudflareWorkersDeploymentUrl )
```